docs/services/data/deploy-model.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Deploy a Model"
+linkTitle: "Deploy Model"
+weight: 60
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "Deploy an image classification model to a robot."
+# SME: Aaron Casas
+---
+
+On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **MODELS** sub-tab.
+
+![The trained model](../img/stars-model.png)
+
+Click the **COPY** button to copy the model information.
```

Comment:
I think you can omit this part. It's only used if someone is using raw JSON in order to get a specific version of a model (power user case), whereas the default flow will be to use the ML Models panel that you describe next in order to grab the latest version of each model.